adafruit_displayio_layout/widgets/icon_animated.py

```diff
@@ -25,12 +25,17 @@
 import time
 from math import pi
 import bitmaptools
-from displayio import TileGrid, Bitmap, Palette
+from displayio import TileGrid, Bitmap, Display, Palette
 import adafruit_imageload
 from adafruit_displayio_layout.widgets.icon_widget import IconWidget
 from adafruit_displayio_layout.widgets.easing import quadratic_easeout as easein
 from adafruit_displayio_layout.widgets.easing import quadratic_easein as easeout
 
+try:
+    from typing import Any, Optional, Tuple
```

Comment:
I think we want the display import to be in this try block after the typing import. 
```
from displayio import Display
```

This way the import will not occur on microcontrollers which don't have `typing` so we will save a bit of RAM by not importing the class that won't be used at run time.